packages/core/interfaces/src/index.ts

```diff
@@ -35,11 +37,8 @@ export interface UseUser
     username: string;
     password: string;
     [x: string]: any;
-  }) => Promise<void>;
-  logout: () => Promise<void>;
-  changePassword: (
-    currentPassword: string,
-    newPassword: string
+  }) => Promise<void>; logout: () => Promise<void>; changePassword: (currentPassword: string,
```

Comment:
please fix this formatting, there is something wrong here

Reply:
fixed